Veriff/Coordinator/CoordinatorImpl.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  CoordinatorImpl.swift
+//  Veriff
+//
+//  Created by Kseniya Zharikova on 25/7/22.
+//
+
+import Foundation
+import UIKit
+
+protocol Coordinator {
+    func start(from viewController: UIViewController, delegate: VeriffCoordinatorDelegate)
+    func didFinishFaceRecognition(result: Result<UIImage,Veriff.ErrorReason>)
```

Comment:
```suggestion
    func didFinishFaceRecognition(result: Result<UIImage, Veriff.ErrorReason>)
```
тут и ниже везде после запятых в перечислении параметров лучше ставить пробелы